test/plugins/osx-apps/index.spec.js

```diff
@@ -12,6 +12,16 @@ const appsList = [
     name: 'App Store',
     filename: 'App Store.prefPane',
     path: '/System/Library/PreferencePanes/AppStore.prefPane'
+  },
+  {
+    name: 'League of Legends',
+    filename: 'League of Legends.app',
+    path: '/Applications/League of Legends.app'
+  },
+  {
+    name: 'AudioBookBinder',
+    filename: 'AudioBookBinder.app',
+    path: '/Applications/AudioBookBinder.app'
```

Comment:
Okay, as I didn't see the file `getAbbr` before I did not know how you get the abbrevations ;)
Oh and I overlooked that this is just the test file X)